### PR TITLE
Add border color commands

### PR DESCRIPTION
**Pull request by fluxchief**
i3s way of setting border colors is really awkward, maybe we should choose a different approach with blocks, eg:

```
border {
    focused {
        border = #FFFFFFFF
        background = #FFFFFFFF
        ...
    }
    unfocused {
        ...
    }
}
```

### CODE REVIEW

File: sway/commands.c [hunk 1]
```diff
@@ -408,6 +417,63 @@ static struct cmd_results *cmd_border(int argc, char **argv) {
...
+
+	uint32_t colors[5];
+	int i;
+	for (i = 0; i < 5; i++) {
+		if (sscanf(argv[i], "#%x", &colors[i]) != 1) {
+			return cmd_results_new(CMD_INVALID, cmd_name, "Invalid color value");
+		}
+	}
+
+	// Colors are in RGB format, add alpha channel
```
Comment (mikkeloscar) [inline sway/commands.c]: It should be possible to optionally define the alpha channel in the config (even though i3 doesn't support that).

Comment (fluxchief) [inline sway/commands.c]: I'd also do this. @SirCmpwn What do you think?
if color >= 0x100000 don't append 0xFF as alpha channel

File: sway/commands.c [hunk 2]
```diff
@@ -45,6 +45,15 @@ static sway_cmd cmd_bar;
 static sway_cmd cmd_bindcode;
 static sway_cmd cmd_bindsym;
 static sway_cmd cmd_border;
+// Border colors, maybe we should not do it the i3 way but
+// create our own config section color {} instead?
```
Comment (mikkeloscar) [inline sway/commands.c]: This is a valid question, but should not be in the source, rather put it in the PR description so we can discuss it. :)

I agree that the syntax for defining colors is very strange considering the rest of the syntax used in i3.

File: sway/commands.c (hunk 1 above)
Comment (mikkeloscar) [inline sway/commands.c]: @fluxchief FYI we already do this for bar colors: https://github.com/SirCmpwn/sway/blob/master/sway/commands.c#L2800-L2827

Comment (fluxchief) [inline sway/commands.c]: Oh, nice, I'll change my code accordingly

**Comment by SirCmpwn** [comment]
LGTM

**Comment by SirCmpwn** [comment]
Regarding a colors block, see #547

**Pull request merged**